Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/SessionWrapper.java

```diff
@@ -11,25 +13,48 @@
     private final RoomWrapper room;
     private final GameWrapper game;
 
+    /**
+     * Intializes the SessionWrapper class.
+     *
+     * @param session The session, not null.
+     */
     public SessionWrapper(Session session) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)